schemas/test/en/test_progress_value_source_section_enabled_hub_complex.json

```diff
@@ -0,0 +1,251 @@
+                            "question": {
+                                "id": "s2-b1-q1",
+                                "title": "Section 2 Question 1",
+                                "description": ["This question always shows. The next question in the section also shows when the answer is not 0"],
+                                "type": "General",
+                                "answers": [
+                                    {
+                                        "id": "s2-b1-q1-a1",
+                                        "mandatory": true,
+                                        "label": "Enter any number",
+                                        "type": "Number"
+                                    }
+                                ]
+                            }
+                        },
+                        {
+                            "type": "Question",
+                            "id": "s2-b2",
+                            "question": {
+                                "id": "s2-b2-q1",
+                                "title": "Section 2 Question 2",
+                                "type": "General",
+                                "description": ["Shows if the answer to the previous question is not 0"],
+                                "answers": [
+                                    {
+                                        "id": "s2-b2-q1-a1",
+                                        "mandatory": true,
+                                        "label": "Enter any number",
+                                        "type": "Number"
+                                    }
+                                ]
+                            },
+                            "routing_rules": [
+                                {
+                                    "when": {
+                                        "!=": [
+                                            {
+                                                "identifier": "s2-b1-q1-a1",
+                                                "source": "answers"
+                                            },
+                                            0
+                                        ]
+                                    },
+                                    "block": "s2-b3"
+                                },
+                                {
+                                    "section": "End"
+                                }
+                            ]
+                        },
+                        {
+                            "type": "Question",
+                            "id": "s2-b3",
+                            "question": {
+                                "id": "s2-b3-q1",
+                                "title": "Section 2 Question 3",
+                                "type": "General",
+                                "description": ["Shows if the answer to the previous question is not 0"],
```

Comment:
shows if the answer to section 2 question 1 is not 0 (but this implies question 2)

Reply:
Updated 👍

Reply:
thanks - just the reopened comment from Kyle outstanding now (the question before this still shows an incorrect description)

Reply:
Updated 👍